src/main/java/org/threadly/litesockets/TCPClient.java

```diff
@@ -206,16 +206,21 @@ protected void doSocketWrite(boolean doLocal) {
 
   @Override
   public ListenableFuture<?> write(final ByteBuffer bb) {
+    return write(new MergedByteBuffers(false, bb));
```

Comment:
I mean we *could* just duplicate the logic, but I dont see an easy way to do it other then that.

Everything else I thought of would involve moving MergedByteBuffers to an interface so we can make s simpler wrapper for single BB's